easybuild/easyconfigs/c/CRISPR-DAV/CRISPR-DAV-2.3.4-foss-2020b.eb

```diff
@@ -0,0 +1,35 @@
+name = 'CRISPR-DAV'
+version = '2.3.4'
+
+homepage = 'https://github.com/pinetree1/crispr-dav/'
+description = """CRISPR-DAV is a pipeline to analyze
+amplicon-based NGS data of CRISPR clones in a high throughput manner."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://github.com/pinetree1/crispr-dav/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = ['crispr-dav-2.3.4-foss2020b_remove_hardcoding.patch']
```

Comment:
The script is not specific to `foss/2020b` at all, so that part should be removed from the patch name.
I would also use `CRISPR-DAV` and `%(version)s`, so:

```python
patches = ['CRISPR-DAV-%(version)s-foss2020b_remove_hardcoding.patch']
```

Don't forget the rename the patch file itself accordingly, and note that there are actually *two* patches with a very similar name currently, one should be removed.

Reply:
correct, renamed